### ISSUE TITLE
Final Project: Topic Selection due 11/5

### ISSUE THREAD

**Issue by Mestrace** [Mestrace/EECS600-deep-learning#1]

**Comment by Mestrace** [comment]
# A2-Nets: Double Attention Networks

- Do a comparison with [SENet (CVPR2018)](https://zpascal.net/cvpr2018/Hu_Squeeze-and-Excitation_Networks_CVPR_2018_paper.pdf)
- Extend the usage to smaller networks (MobileNet [Original](https://arxiv.org/abs/1704.04861) // [V2](https://zpascal.net/cvpr2018/Sandler_MobileNetV2_Inverted_Residuals_CVPR_2018_paper.pdf), [ShuffleNet](http://openaccess.thecvf.com/content_cvpr_2018/CameraReady/0642.pdf)

## Some Discussions
- https://wzwhit.github.io/2019/07/17/A-2-Nets-Double-Attention-Networks/
- https://juejin.im/entry/5d649aaaf265da03b31bdcaa
- https://blog.csdn.net/qq_37014750/article/details/83989334
- https://www.zhihu.com/question/321294274

**Comment by Mestrace** [comment]
# DropMax: Adaptive Variational Softmax

- Tune on the dropout probability

## Related Works
-Using different function to determine the dropout probability. [Link](https://www.cs.cmu.edu/~epxing/Class/10715/project-reports/DuyckLeeLei.pdf)

**Comment by Mestrace** [comment]
# Co-teaching: Robust Training of Deep Neural Networks with Extremely Noisy Labels
- Easy to implement (existing implementation available)
- As per the original reviewers
    > Weakness: - It would have been interesting a test with a bigger, state of the art network for classification on ImageNet dataset. Even AlexNet would have been interesting to see
    
    Could do experiments on more complicated networks / dataset

## Procedures

### Noisy Label Generation
- Add random noise to the data
- Use other recognition model to generate label for given data
    - What model to use?
    - Can we control the noisy rate?
    - Can we interpret why this model makes mistakes (generate erroneous labels), and can our model deal with these noise?

### Model Training
- Use more complex models, as suggested by the reviews
- Use different model for teaching
- Use more than 2 models
    - Cyclic structure: pass selected batch samples to the next model
    - Pool: Every model gets a different batch; selected samples got into a pool; every model gets a partition from the pool for training.

### Answering the Questions
- What is the problem you are addressing and why is it pertinent?
- What research have you done into the context of this problem?
- What is the data that you will use? How did/will you get it?
- How are you planning on implementing your solution?
- What metrics are you using to evaluate your results?
